### ISSUE TITLE
Reframing the question, Is there any special access needed to use protected API's?

### ISSUE THREAD

**Issue by Vmit2** [cowinapi/developer.cowin#176]

**Comment by deepgandhi** [comment]
Yes, you have to get "x-api-key" for production server for generating and verifying OTP, which in turn generate token, which can be used for Bearer Authorization in production server for all other protected apis.

**Comment by Vmit2** [comment]
> Yes, you have to get "x-api-key" for production server for generating and verifying OTP, which in turn generate token, which can be used for Bearer Authorization in production server for all other protected apis.

And where should I get the x-api-key for production,  I saw there is x-api-key only for demo server.

**Comment by deepgandhi** [comment]
> > Yes, you have to get "x-api-key" for production server for generating and verifying OTP, which in turn generate token, which can be used for Bearer Authorization in production server for all other protected apis.
> 
> And where should I get the x-api-key for production,  I saw there is x-api-key only for demo server.

It is not yet available, but you can use cowin.gov.in for generating Bearer token and use other protected api by using that token.

**Comment by siddhesh1770** [comment]
> > Yes, you have to get "x-api-key" for production server for generating and verifying OTP, which in turn generate token, which can be used for Bearer Authorization in production server for all other protected apis.
> 
> And where should I get the x-api-key for production, I saw there is x-api-key only for demo server.

Right now it  is available for government entities like Aarogya Setu, Umang etc.
But if you are employee of Municipal corporation, district office or any other equivalent government body you can get access to Protected API for integrating in your Website or App by Contacting Ministry of Health
